src/data/content/design-system/fondamenti/technical-template.yaml

```diff
@@ -170,4 +170,74 @@ components:
           accordionUrl: https://italia.github.io/bootstrap-italia/docs/organizzare-gli-spazi/bordi/
           accordionOpen: false
           accordionShow: true
-          
+
+        - name: TextImageCta
+          title: Varianti di raggio
+          text: |
+            _In stesura_ 
+        - name: Table
+          title: Esempio tabella
+          headingLevel: 4
```

Comment:
Even if I don't like this at all, I'd use "`h4`" instead of just `4`, so it's clear what is going to do. 

We have other instances of this in other components, so we'd better fix it everywhere and think about headings in a sane and non error-prone way.

Commenting on this to have a base for opening a new issue.

Reply:
Open an issue, we don't resolve it here.